src/lib.rs

```diff
@@ -38,7 +38,7 @@ use core::str::FromStr;
 #[cfg(feature = "std")]
 use std::error::Error;
 
-use traits::{Inv, Num, One, Zero};
+use traits::{Inv, Num, One, Zero, MulAdd};
```

Comment:
Please run your changes through `rustfmt` / `cargo fmt`. It will sort these imports, for one, but I expect some of your other code will change formatting too.